frontend/src/services/AuthService.ts

```diff
@@ -119,8 +121,11 @@ const removeFamUser = () => {
     localStorage.removeItem(CURRENT_SELECTED_APPLICATION_KEY);
 }
 
-const storeFamUser = (famLoginUser: FamLoginUser | null | undefined) => {
+async function storeFamUser(famLoginUser: FamLoginUser | null | undefined) {
     state.value.famLoginUser = famLoginUser;
+    state.value.famLoginUser!.accesses = (
+        await AdminMgmtApiService.adminUserAccessesApi.adminUserAccessPrivilege()
+    ).data.access;
```

Comment:
also I wonder if you tested the case when this api returns error or it's down, what you will get here, you might need to consider the error handling.

Reply:
If there is an error should it be a toast or a message?